src/features/contacts/add/index.tsx

```diff
@@ -0,0 +1,99 @@
+import { useState } from "react";
+import {
+  Breadcrumbs,
+  Link,
+  Typography,
+  AlertColor,
+  Backdrop,
+  CircularProgress,
+} from "@mui/material";
+import { NavigateNext } from "@mui/icons-material";
+import { ContactCreateDto, ContactDetailsDto } from "lib/network/swagger-client";
+import {
+  ModuleContainer,
+  ModuleHeaderContainer,
+  ModuleHeaderSubtitleContainer,
+  ModuleHeaderTitleContainer,
+} from "components/module";
+import { rootRoute, CoreModule, getCoreModuleRoute } from "lib/router";
+import { GhostLink } from "components/ghost-link";
+import { useRequestContext } from "providers/request-provider";
+import { ContactForm } from "../form";
+import { CustomizedSnackbar } from "components/snackbar";
+
+export const ContactAdd = () => {
+  const { client } = useRequestContext();
+
+  const [contact, setContact] = useState<ContactDetailsDto>({ email: "" });
+
+  const [isSaving, setIsSaving] = useState(false);
+
+  const [snackBarParams, setParams] = useState({
+    message: "",
```

Comment:
better to make it [snackBarParams, setSnackBarParams],
it's more convenient for other developers even if it's long

Reply:
Updated.